AdvLoggerPkg/Library/AdvancedLoggerHdwPortLib/AdvancedLoggerHdwPortLib.c

```diff
@@ -58,12 +58,5 @@ AdvancedLoggerHdwPortWrite (
   IN UINTN  NumberOfBytes
   )
 {
-  UINTN  NumberReturned;
-
-  NumberReturned = NumberOfBytes;
-  if (DebugLevel & PcdGet32 (PcdAdvancedLoggerHdwPortDebugPrintErrorLevel)) {

```

Comment:
Since the intention is to check debug level in the upper layer only and "DebugLevel" argument is not used anymore in this API, may be "DebugLevel" argument can be removed altogether from this API.